homeassistant/components/rachio/binary_sensor.py

```diff
@@ -52,6 +58,11 @@ def _create_entities(hass: HomeAssistant, config_entry: ConfigEntry) -> list[Ent
     for controller in person.controllers:
         entities.append(RachioControllerOnlineBinarySensor(controller))
         entities.append(RachioRainSensor(controller))
+    entities.extend(
+        RachioHoseTimerBattery(valve, base_station.coordinator)
+        for base_station in person.base_stations
+        for valve in base_station.coordinator.data.values()
+    )
```

Comment:
This is ok now, but if we get more of these, we should switch to BinarySensorEntityDescription.

Reply:
Hold off one sec on merging

Reply:
I was thinking the same thing, will switch if I add any more.